### PR TITLE
Exclude ES benchmarks

### PR DESCRIPTION
**Pull request by mikethebeer**
* Client API has changed so don't execute ES benchmarks to be compatible
  with old versions of Crate.

* added jitpack repo to solve Crate Client dependency

* point nightly benchmark job in jenkins to `es-bench-upgrade` repo to get successfully passed benchmark results.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,18 +4,40 @@ apply plugin: 'idea'
...
 
 dependencies {
+
     testCompile 'junit:junit:4.12'
     testCompile 'com.carrotsearch:junit-benchmarks:0.7.2'
     testCompile 'com.h2database:h2:1.3.173'
-    testCompile 'io.crate:crate-client:' + project.crateVersion
 
-    testCompile 'io.crate:crate-testing:0.3.1'
+    //testCompile 'com.github.crate.crate:crate-client:' + project.tag
```
Comment (chaudum) [inline build.gradle]: remove

File: build.gradle [hunk 2]
```diff
@@ -103,21 +121,7 @@ sourceSets {
     }
 }
 
-def jvmTestFlags = ['-ea']
```
Comment (chaudum) [inline build.gradle]: is there a reason why you removed assertions?

File: src/test/java/io/crate/stress/AbstractIntegrationStressTest.java [hunk 3]
```diff
@@ -90,13 +89,13 @@ public static void init() {
...
     @Before
     public void delegateToPrepareFirst() throws Exception {
-        if (esClient == null) {
-            esClient = new TransportClient(ImmutableSettings.builder().put("cluster.name", CLUSTER_NAME).build());
-            for (CrateTestServer server : testCluster.servers()) {
-                InetSocketTransportAddress serverAdress = new InetSocketTransportAddress(server.crateHost(), server.transportPort());
-                esClient.addTransportAddress(serverAdress);
-            }
-        }
+//        if (esClient == null) {
```
Comment (chaudum) [inline src/test/java/io/crate/stress/AbstractIntegrationStressTest.java]: why only commented out? can be removed

File: build.gradle (hunk 2 above)
Comment (mikethebeer) [inline build.gradle]: @chaudum --> https://github.com/crate/crate-stresstest/pull/22/commits/5ae5441210e27a96dbd5731f42da4e7b651d738f

**Comment by chaudum** [comment]
ack

**Pull request merged**